src/main/java/domain/LottoNumber.java

```diff
@@ -0,0 +1,47 @@
+package domain;
+
+import java.util.Objects;
+
+public class LottoNumber {
+
+    private static final int MIN_NUMBER_VALUE = 1;
+    private static final int MAX_NUMBER_VALUE = 45;
+
+    private final int value;
+
+    private LottoNumber(final int value) {
+        this.value = value;
+    }
+
+    public static LottoNumber valueOf(final int value) {
+        validateRange(value);
+        return new LottoNumber(value);
+    }
+
+    private static void validateRange(final int value) {
+        if (value < MIN_NUMBER_VALUE || value > MAX_NUMBER_VALUE) {
+            throw new IllegalArgumentException("1에서 45 범위에서 벗어났습니다.");
```

Comment:
아래와 같이 파라미터도 포함해주시면 디버깅시에 좀 더 빠르게 문제점을 찾을 수 있겠네요 😄 
```suggestion
            throw new IllegalArgumentException(String.format("1에서 45 범위에서 벗어났습니다. %d", value));
```